css/App.css

```diff
@@ -0,0 +1,38 @@
+iframe {
+  display: none;
+}
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
+label,
+button {
+  cursor: pointer;
+}
+
+.error {
+  color: red;
+}
+
+.TodoInfo {
+  width: -moz-max-content;
+  width: max-content;
```

Comment:
The `width: -moz-max-content;` property is outdated and not needed when using `width: max-content;`. Modern browsers support `max-content` without the need for vendor prefixes. You can safely remove the `-moz-max-content` line.